lighthouse-core/computed/page-dependency-graph.js

```diff
@@ -178,6 +190,17 @@ class PageDependencyGraph {
       cpuNode.addDependent(networkNode);
     }
 
+    /** @param {CPUNode} cpuNode @param {string|undefined} frameId */
```

Comment:
can you add a function description ("if the node has an associated frameId, then...")